docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,428 @@
+  "supportedDevices": [
+    "raspberrypi",
+    "raspberrypi2",
+    "raspberrypi3"
+  ]
+}
+```
+
+Describe an image with any of the following properties:
+
+##### `displayName (String)`
+
+The human-friendly name of the image.
+
+##### `version (String)`
+
+The version of the image.
+
+##### `url (String)`
+
+The main url of the image.
+
+##### `releaseNotesUrl (String)`
+
+The url to the image's release notes or CHANGELOG. This URL will take
+precedence over `.meta/release-notes.txt` if defined.
+
+##### `releaseDate (String)`
+
+The release date timestamp. The date should conform to [ISO 8601][iso8601]
+standard.
+
+##### `checksumType (String)`
+
+The checksum type. The current possible values are: `sha1`, `crc32`, and `md5`.
+
+See the `.meta/checksum` file.
+
+##### `recommendedDriveSize (Number)`
+
+The minimum recommended drive size to flash this image, in bytes.
+
+The use case for this option is that while a drive might be large enough to
+contain the image, it might not be large enough to deliver a good experience
+when actual using the application or operating system contained in the image.
+
+##### `supportedDevices (String[])`
+
+An array of the device slugs this image supports.
+
+##### `updateUrl (String)`
+
+The url where the client can request the latest version of the image.
+
+The way this works is that you provide a generic url that redirects to the
+latest image resource. The client will follow all redirects until it finds the
+final image, and will check (using the `etag` property) if the resource matches
+the currently selected image.
+
+If the ETags don't match, the client may point the user to download the new one
+image.
+
+If the HTTP response from `updateUrl` doesn't contain an ETag this mechanism is
+ignored.
+
+##### `etag (String)`
+
+The [HTTP Etag][etag] of the current image.
+
+Clients may use this to check if the image resolved by `updateUrl` is different
+to the one we have locally.
+
+##### `expiryDate (String)`
+
+The timestamp to determine the expiration date of the image. The date should
+conform to [ISO 8601][iso8601] standard.
+
+Clients may use this information to refuse flashing the image.
+
+#### Extensions
+
+Each of the entities described above accept an object property called
+`extensions`, which you can use to provide custom properties to the manifests,
+in a way that are specific to your application. It is encouraged that you put
+those custom properties inside the `extensions` namespace to avoid potential
+clashes with other properties that may be introduced in the future.
+
+For example:
+
+```json
+{
+  "publisher": {
+    ..
+    "extensions": {
+      ..
+    }
+    ..
+  },
+  "device": {
+    ..
+    "extensions": {
+      ..
+    }
+    ..
+  },
+  "image": {
+    ..
+    "extensions": {
+      ..
+    }
+    ..
+  }
+}
+```
+
+### `.meta/publisher.svg`
+
+An [`svg`][svg] logo of the publisher.
+
+### `.meta/device.svg`
+
+An [`svg`][svg] logo of the device.
```

Comment:
The path to this should be pointed to from the device section (with `device.svg` just being a recommended default)